modules/cloudaccess/examples/complete/aws-to-aws.tfvars.sample

```diff
@@ -0,0 +1,8 @@
+# Resource suffix is optional, specify this if you need to keep role definitions unique amongst
+# multiple workspaces using the same AWS Account.
+resource_suffix                 = "-app1"
```

Comment:
this isn't needed by default on AWS - only on azure where role definitions need to be tenant-wide unique

Reply:
I'll drop it